src/pages/NotFound.vue

```diff
@@ -10,9 +10,19 @@
 
 <script>
 import HomeLayout from '@/layouts/Home';
+
 export default {
   name: 'NotFound',
   components: { HomeLayout },
+  data() {
```

Comment:
I don't understand why you changed `NotFound.vue`. Were you testing things and just forgot to remove this? :thinking:

Reply:
lol seems like I did forget to take that out😂